### PR TITLE
travis config file for test and heroku deployment

### PR DESCRIPTION
**Pull request by shamnadps**

### CODE REVIEW

**Comment by shamnadps** [comment]
@walokra yes docker container:push web does the docker build.

File: backend/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: generic
```
Comment (StormBlade97) [inline backend/.travis.yml]: typescript and javascript here maybe? IDK about this script but looks like a specificity could help

File: backend/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+sudo: required
+services:
+- docker
+script:
+- cd backend
```
Comment (StormBlade97) [inline backend/.travis.yml]: cant docker handle this?

Comment (shamnadps) [reply backend/.travis.yml]: what do you mean by this?.

**Pull request merged**